bin/setup

```diff
@@ -24,16 +24,21 @@ chdir APP_ROOT do
 
   puts "\n== Copying sample files =="
   unless File.exist?('config/database.yml')
-    cp 'config/database.yml.sample', 'config/database.yml'
+    FileUtils.cp 'config/database.yml.sample', 'config/database.yml'
   end
 
   puts "\n== Preparing database =="
-  system! 'bin/rails db:setup'
+  db_exists = system('psql -o /dev/null -q -n -c "select 1" PracticalDeveloper_development')
+  if db_exists
+    puts 'bin/rails db:migrate'
```

Comment:
`puts`, not `system!`?

Reply:
hahha sorry, this slipped in a botched save from the editor! Fixed it a tested it multiple times!